Shared/Samples/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioningView.swift

```diff
@@ -21,27 +21,15 @@ struct ShowDeviceLocationUsingIndoorPositioningView: View {
     @StateObject private var model = Model()
     /// The error shown in the error alert.
     @State private var error: Error?
-    /// This is the text description of the current floor that is displayed.
-    @State private var currentFloorText: String = ""
-    /// This is the text description of the data source that is displayed.
-    @State private var sourceText: String = ""
-    /// This is the text description of the number sensor of the data that is displayed.
-    @State private var sensorText: String = ""
-    /// This is the text description of the horizontal accuracy data that is displayed.
-    @State private var accuracyText: String = ""
-    
+
```

Comment:
Notice this change? It indicates that the whitespace on the line changed. When reviewing your code before committing (which you should do every time!), pay careful attention to whitespace-only changes. In this case, the whitespace was correct before and now it is not. This is something to catch and fix before committing. Please revert.

Reply:
Okay, I think it is fixed